config/routes.rb

```diff
@@ -1,66 +1,15 @@
 Rails.application.routes.draw do
-  devise_for :users
-  resources :genres
-
-  resources :artists
-
-  resources :users
-
   root 'search#index'
   get '/search', to: 'search#show'
 
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
-
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  get 'web', to: 'web#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.